jobs/director/spec

```diff
@@ -441,6 +441,9 @@ properties:
   agent.env.bosh:
     description: Base env for agent
     default: {}
+  agent.agent_wait_timeout:
+    description: optional agent wait timeout setting, default 600s
+    default: 600
```

Comment:
which option do u prefer? I think keep the default value would be a safer option to make sure it always have a value to avoid the need to consider nil|null value in code base or templating.

Reply:
okay had some internal discussion and we can just keep it as is.